tox.ini

```diff
@@ -18,6 +19,12 @@ covercmd =
       --cov-append \
       --cov-config {toxinidir}/.coveragerc \
       core/unit_tests
+    py.test --quiet \
```

Comment:
I'm not sure I understand why in `.travis.yml` you do `(cd storage && tox -e cover)` but you also do coverage for storage here. Does it not end up running the same tests twice?